### DISCUSSION TITLE
Disable metadata/schema changes via cloud console

### DISCUSSION THREAD

**Discussion by erquhart** [hasura/graphql-engine#8943] [Question] [answered]
If I manage my Hasura cloud project via Git deployments, I'll want to ensure that changes are not made directly through the cloud console. The docs recommend disabling the console on the local server for this reason, but I don't see a way to disable the cloud console. Hasura cloud disallows use of the environment variable that controls this in graphql-engine.

Any way to accomplish this?

**Comment by adas98012** [comment, marked as answer]
You are right, the console cannot be disabled in cloud, in the current product setup.
